working/wildcards/feature-specification.md

```diff
@@ -244,6 +255,39 @@ class C {
 }
 ```
 
+### Super parameters
+
+An occurrence of `super._` as a declaration of a formal parameter in a
+constructor is a compile-time error.
```

Comment:
Can you also help me visualize this, and maybe the following below, by adding a little code snippet/example?
The words make sense, I just learn better with examples.

Reply:
Done. This brings out a slight inconvenience, by the way. I added a commentary paragraph about it.